S23/mvr9999/AllTestCasesToLowerCase.java

```diff
@@ -0,0 +1,100 @@
+package com.firstspringproject.Controllers;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.*;
+
+public class ToLowerCase {
+    
+    String turkish_lang = "tr";

```

Comment:
I feel that naming conventions for attribute names in camelCase looks better rather than separating with "_"
Ex:  turkish_lang to turkishLang

Reply:
noted!